Assets/HoloToolkit-Examples/SharingWithUNET/Scripts/UNetAnchorManager.cs

```diff
@@ -205,14 +205,14 @@ public void WaitForAnchor()
         /// Attempts to attach to  an anchor by anchorName in the local store..
         /// </summary>
         /// <returns>True if it attached, false if it could not attach</returns>
-        private bool AttachToCachedAnchor(string AnchorName)
+        private bool AttachToCachedAnchor(string anchorName)
```

Comment:
@darax it would be nice to make the `UNetAnchorManager` inherit from the `WorldAnchorManager`, but I wasn't sure where to start.  I noticed this class inherits from `NetworkBehavior`.